x/liquidstaking/keeper/rebalancing_test.go

```diff
@@ -286,6 +273,200 @@ func (s *KeeperTestSuite) TestRebalancingCase1() {
 	s.RequireNetAmountStateZero()
 }
 
+func (s *KeeperTestSuite) TestRebalancingConsecutiveCase() {
+	_, valOpers, _ := s.CreateValidators([]int64{
+		1000000, 1000000, 1000000, 1000000, 1000000,
+		1000000, 1000000, 1000000, 1000000, 1000000,
+		1000000, 1000000, 1000000, 1000000, 1000000})
+	s.ctx = s.ctx.WithBlockHeight(100).WithBlockTime(utils.ParseTime("2022-03-01T00:00:00Z"))
+	params := s.keeper.GetParams(s.ctx)
+	params.UnstakeFeeRate = sdk.ZeroDec()
+	params.MinLiquidStakingAmount = sdk.NewInt(10000)
+	s.keeper.SetParams(s.ctx, params)
+	s.keeper.UpdateLiquidValidatorSet(s.ctx)
+
+	stakingAmt := sdk.NewInt(10000000000000)
+	s.fundAddr(s.delAddrs[0], sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, stakingAmt)))
+	// add active validator
+	params.WhitelistedValidators = []types.WhitelistedValidator{
+		{ValidatorAddress: valOpers[0].String(), TargetWeight: sdk.NewInt(1)},
+		{ValidatorAddress: valOpers[1].String(), TargetWeight: sdk.NewInt(1)},
+		{ValidatorAddress: valOpers[2].String(), TargetWeight: sdk.NewInt(1)},
+		{ValidatorAddress: valOpers[3].String(), TargetWeight: sdk.NewInt(1)},
+		{ValidatorAddress: valOpers[4].String(), TargetWeight: sdk.NewInt(1)},
+		{ValidatorAddress: valOpers[5].String(), TargetWeight: sdk.NewInt(1)},
+		{ValidatorAddress: valOpers[6].String(), TargetWeight: sdk.NewInt(1)},
+		{ValidatorAddress: valOpers[7].String(), TargetWeight: sdk.NewInt(1)},
+	}
+	s.keeper.SetParams(s.ctx, params)
+	reds := s.keeper.UpdateLiquidValidatorSet(s.ctx)
+	s.Require().Len(reds, 0)
+
+	newShares, bTokenMintAmt, err := s.keeper.LiquidStake(s.ctx, types.LiquidStakingProxyAcc, s.delAddrs[0], sdk.NewCoin(sdk.DefaultBondDenom, stakingAmt))
+	s.Require().NoError(err)
+	s.Require().Equal(newShares, stakingAmt.ToDec())
+	s.Require().Equal(bTokenMintAmt, stakingAmt)
+	// assert rebalanced
+	reds = s.keeper.UpdateLiquidValidatorSet(s.ctx)
+	s.Require().Len(reds, 0)
+	s.Require().Equal(s.redelegationsErrorCount(reds), 0)
```

Comment:
Just a quick note: `require.Equal` has a signature of `func(t, expected, actual, ...)`, so we need to change this to:
```suggestion
	s.Require().Equal(0, s.redelegationsErrorCount(reds))
```

Reply:
Sure, Thanks, Let me fix after on other PR